.gitignore

```diff
@@ -1,4 +1,3 @@
 /node_modules
 .DS_STORE
-.gitignore
-.env
\ No newline at end of file
+.gitignore
```

Comment:
Not sure why `.env` is being removed, can you add back?

Reply:
I can’t push my merge commit "main -> main (push declined due to repository rule violations)". This may be why Github is still flagging some merge conflicts in the Opportunities and OpportunityDetails files. How do you think?